cms/templates/js/course-outline.underscore

```diff
@@ -57,7 +60,16 @@ if (staffOnlyMessage) {
         addStatusMessage(messageType, messageText);
     }
 
-    if (hasPartitionGroups) {
+    if (selectedPartitionIndex !== -1 && !isNaN(selectedPartitionIndex) && xblockInfo.isVertical()) {
```

Comment:
I don't really know javascript that well, but is !isNan(selectedPartitionIndex) the cleanest way to do this? Is selectedPartitionIndex ever below 0?